packages/chrome/src/elements/Chrome.tsx

```diff
@@ -5,24 +5,26 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { HTMLAttributes, useMemo } from 'react';
+import React, { HTMLAttributes, useMemo, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { ThemeProps, DefaultTheme } from 'styled-components';
 import { readableColor } from 'polished';
-import { withTheme, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { withTheme, DEFAULT_THEME, getColor, useDocument } from '@zendeskgarden/react-theming';
 import { ChromeContext } from '../utils/useChromeContext';
 import { StyledChrome } from '../styled';
 
 interface IChromeProps extends HTMLAttributes<HTMLDivElement> {
   /** Apply a custom hue to chrome navigation */
   hue?: string;
+  /** Apply a fixed position to the <html> element */
```

Comment:
Please update this description. Something like `Prevent fixed positioning from being applied to the <html> element`

Reply:
☝️

Reply:
Doh. In some universe, I read this as update the PR description.

Reply:
😅